test/wallaby/dsl_test.exs

```diff
@@ -166,4 +166,24 @@ defmodule Wallaby.DSLTest do
     assert element
     Application.put_env(:wallaby, :base_url, nil)
   end
+
+  test "taking a screenshot", %{session: session, server: server} do
+    path =
+      session
+      |> visit(server.base_url)
+      |> take_screenshot
+
+    assert File.exists? path
+    File.rm_rf! path
```

Comment:
It'd be nice to cleanup the /screenshots directory as well but I wasn't sure how to get the app root